packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/descriptions/ColumnDescription.java

```diff
@@ -96,6 +109,10 @@ public static final class Builder {
 
         private Function<VariableManager, List<Object>> semanticElementsProvider;
 
+        private Function<VariableManager, Integer> initialWidthProvider = variableManager -> 150;
+
+        private Predicate<VariableManager> isResizablePredicate = variableManager -> false;
```

Comment:
Don't make choices for the specifier, ask for the value instead

Reply:
Also consider that if the specifier returns something like -1 it should be ignored on the frontend. Given that some columns may have a large or small label the specifier may not have any idea of a proper size.